### ISSUE TITLE
⭐FEAT:

### ISSUE THREAD

**Issue by ayushichoudhary-19** [ArslanYM/StarterHive#357]
**Is your feature request related to a problem? Please describe.**
The current placement of the "movetotop" button is limited to contributor and docs pages. This presents an inconvenience when navigating the project, as there are times when users may need to return to the top of the project's page for various reasons.

**Describe the solution you'd like**
I would like to propose adding a "movetotop" button on the project's page as well. This would enhance user experience by providing a convenient way to quickly scroll back to the top of the page regardless of where they are within the project's content.

**Describe alternatives you've considered**
An alternative solution could be using the browser's built-in scroll functionality, but this may not be as user-friendly and efficient as a dedicated "movetotop" button.

**Additional context**
No additional context or screenshots are required for this feature request as it is a straightforward usability improvement.

---
**Note:**
I would like to work on this issue myself.

**Comment by amanrajrana** [comment]
Hi @ayushichoudhary-19, This website already has "movetotop" Button functionality on some pages (e.g.: [/contibutores](https://starter-hive.vercel.app/contributors) & [/docs](https://starter-hive.vercel.app/docs)). So make sure you are not building it from scratch. you have to just implement the already built component name `fab` to the project page.

**Comment by ayushichoudhary-19** [comment]
Yes, I am aware of that and will keep it in mind. Thanks for clarification.

**Comment by ArslanYM** [comment]
@ayushichoudhary-19 Do you want to work on this?

**Comment by ayushichoudhary-19** [comment]
> @ayushichoudhary-19 Do you want to work on this?

Yes!

**Comment by ArslanYM** [comment]
@ayushichoudhary-19 Assigned to you.
